package/MDAnalysis/analysis/dssp.py

```diff
@@ -0,0 +1,279 @@
+from .base import AnalysisBase
```

Comment:
Add introductory text for the module. Describe what it does, include citations to DSSP.

Acknowledge pydssp (author, version, license, and link).

You can add an example section here instead of overloading the doc string.

Reply:
Create two doc sections: one for user-code (`DSSP`), the other for helper functions such as `get_hbond_map`.

Reply:
Could you point me to the best example of how it's done somewhere in the library?

For now, I used `.. autofunction::` for the `get_hbond_map`, and mostly focused on user documentation here.